openedx_learning/core/publishing/models.py

```diff
@@ -32,7 +33,7 @@ class LearningPackage(models.Model):
 
     uuid = immutable_uuid_field()
     key = key_field()
-    title = models.CharField(max_length=1000, null=False, blank=False)
+    title = case_insensitive_char_field(max_length=500, blank=False)
```

Comment:
I can't get the publishing API tests to pass on MySQL? Errors are:

```
FAILED tests/openedx_learning/core/publishing/test_api.py::CreateLearningPackageTestCase::test_auto_datetime - django.db.utils.OperationalError: (1366, "Incorrect string value: '\\xF0\\x9F\\x94\\xA5' for column 'title' at row 1")
FAILED tests/openedx_learning/core/publishing/test_api.py::CreateLearningPackageTestCase::test_normal - django.db.utils.OperationalError: (1366, "Incorrect string value: '\\xF0\\x9F\\x94\\xA5' for column 'title' at row 1")
```

I've configured the tests to run against a mysql docker container running with `--character-set-server=utf8mb4 --collation-server=utf8mb4_bin`, and everything on the test database _looks_ like it should be ok, but still the tests fail.. am I missing something?

cf https://stackoverflow.com/q/10957238

```
mysql> use oel_test;
mysql> show full columns from oel_publishing_learningpackage;
+---------+--------------+--------------------+------+-----+---------+----------------+---------------------------------+---------+
| Field   | Type         | Collation          | Null | Key | Default | Extra          | Privileges                      | Comment |
+---------+--------------+--------------------+------+-----+---------+----------------+---------------------------------+---------+
| id      | bigint(20)   | NULL               | NO   | PRI | NULL    | auto_increment | select,insert,update,references |         |
| uuid    | char(32)     | utf8mb4_bin        | NO   | UNI | NULL    |                | select,insert,update,references |         |
| key     | varchar(500) | utf8mb4_bin        | NO   | UNI | NULL    |                | select,insert,update,references |         |
| title   | varchar(500) | utf8mb4_unicode_ci | NO   |     | NULL    |                | select,insert,update,references |         |
| created | datetime(6)  | NULL               | NO   |     | NULL    |                | select,insert,update,references |         |
| updated | datetime(6)  | NULL               | NO   |     | NULL    |                | select,insert,update,references |         |
+---------+--------------+--------------------+------+-----+---------+----------------+---------------------------------+---------+
6 rows in set (0.00 sec)
mysql> show variables like 'character%';
+--------------------------+----------------------------+
| Variable_name            | Value                      |
+--------------------------+----------------------------+
| character_set_client     | utf8mb4                    |
| character_set_connection | utf8mb4                    |
| character_set_database   | utf8mb4                    |
| character_set_filesystem | binary                     |
| character_set_results    | utf8mb4                    |
| character_set_server     | utf8mb4                    |
| character_set_system     | utf8                       |
| character_sets_dir       | /usr/share/mysql/charsets/ |
+--------------------------+----------------------------+
```

(That `character_set_system: utf8` doesn't seem to be the problem, cf https://stackoverflow.com/a/55957424)

Reply:
Ah cool -- I was using the [`DATABASES["TEST"]` stanza](https://docs.djangoproject.com/en/3.2/ref/settings/#std-setting-DATABASE-TEST), not `DATABASES["OPTIONS"]`. That worked!